src/litserve/specs/openai.py

```diff
@@ -395,9 +395,8 @@ def populate_context(self, context, request):
 
     def decode_request(
         self, request: ChatCompletionRequest, context_kwargs: Optional[dict] = None
-    ) -> List[Dict[str, str]]:
-        # returns [{"role": "system", "content": "..."}, ...]
-        return [el.model_dump(by_alias=True, exclude_none=True) for el in request.messages]
+    ) -> ChatCompletionRequest:
```

Comment:
maybe have an argument like `full_response: bool = False` that would enable it but still wint break existing behaviour?